### ISSUE TITLE
508-defect-4 [COGNITION]: COVID Vaccine Trials - 711 link aria-label repeats "TTY" for screen readers

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#18274]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a deaf-blind user who might use a screen reader and TTY services, it could be confusing to hear "TTY, TTY seven one one, link" when parsing our TTY link text.

## Details

The TTY link is duplicating the text TTY in the `aria-label` attribute. The aria label should only read the "seven one one" text, since TTY is presented outside the link. Screenshot and code snippet attached below.

## Acceptance Criteria

- [ ] 711 link's `aria-label` is updated
- [ ] Screen readers read "seven one one, link" when the TTY link receives keyboard focus

## Environment

* https://staging.va.gov/coronavirus-research

## Proposed Solution (if known)

```diff
<p>
  ...
  If you have hearing loss, call TTY:
- <a aria-label="TTY. 7 1 1." href="tel:711">711</a>.
+ <a aria-label="7 1 1." href="tel:711">711</a>.
</p>
```

## WCAG or Vendor Guidance (optional)

* [Name, Role, Value: Understanding SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)

## Screenshots or Trace Logs

<img width="1115" alt="Screen Shot 2021-01-07 at 5 13 48 PM" src="https://user-images.githubusercontent.com/934879/103955862-e77b3180-510c-11eb-93f1-759c8ed6f597.png">

**Comment by 1Copenut** [comment]
Verified the fix has been deployed and working as expected. Moving to close.

**Comment by Mottie** [comment]
All fixes have been merged and should be available in staging soon.

**Comment by Mottie** [comment]
@joshkimux I think we can close this ticket... "TTY" is no longer in the Telephone `aria-label`
